frontend/src/AlertsManagement/AlertsList.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import Checkbox from '@material-ui/core/Checkbox';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import Tooltip from '@material-ui/core/Tooltip';
+
+class AlertsList extends Component {
+  render() {
+    const alerts = this.props.alerts;
```

Comment:
nit: `{ alerts } = this.props;` (?)

Reply:
I don't think you can do that...?

Reply:
oh, i thought it would be the same as destructuring with states. 
my bad! sorry for pointing this out...

Reply:
it's ok <3